### ISSUE TITLE
Install nw fails on mac m1

### ISSUE THREAD

**Issue by utenma** [nwjs/npm-installer#83]
looks like there are no sdk binaries for macos with m1 processors, neither i except any date for them to be available, just posting the fact.

```
npm ERR! command failed
npm ERR! command sh -c node scripts/install.js
npm ERR! Response code 404 (Not Found)
```

**Comment by Seiikatsu** [comment]
Same issue here, for now I used:
`npm_config_nwjs_process_arch=x64 npm install nw`
to install the x64 binary for macos.

**Comment by insanj** [comment]
@Seiikatsu Thank you, this also worked for me without any issues 👍

**Comment by saulpalv** [comment]
looks like it works now by default

**Comment by ayushmanchhabra** [comment]
Closing since no longer an issue. MacOS ARM binaries have been available since v0.77.0
